src/app/screens/PeerlyScreens/ProfileDetailScreen/profile.hooks.ts

```diff
@@ -5,9 +5,9 @@ import toast from '../../../utils/toast';
 
 import {getProfileDetails} from '../../../services/PeerlyServices/profileDetail';
 
-export function useGetProfileDetails() {
+export function useGetProfileDetails(userId?: number) {
   const {data, isLoading, isFetching, isSuccess, isError} = useQuery({
-    queryKey: ['profile_icon'],
+    queryKey: userId ? ['profile_icon', userId] : ['profile_icon'],
```

Comment:
`queryKey: ['profile_icon', userId]`

Should also work in this case.

Reply:
Made the chagnes